tests/test_action_checks.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+
+# Initialize error counter
+errors=0
+
+# Function to simulate the "Check if review is requested" step
+check_review_requested() {
+  PR_TITLE="$1"
+  
+  if [[ "$PR_TITLE" =~ ^(no|skip)(-|\s)?review|skip(-|\s)?code(-|\s)?review|otter(ai)?(-|\s)?skip|otter(-|\s)?restricted ]]; then
```

Comment:
🔍 Consider using a more explicit regex pattern for clarity. This can help future maintainers understand the conditions better.